public/src/components/audio.js

```diff
@@ -0,0 +1,9 @@
+import _ from '../../lib/utils'
+import App from '../app'
+let d = React.DOM
+
+export default React.createClass({
+  render() {
```

Comment:
http://stackoverflow.com/questions/25986439/reactdom-getdomnode-instead-of-jquery